safe_transaction_service/history/serializers.py

```diff
@@ -757,7 +757,9 @@ class SafeCreationInfoResponseSerializer(serializers.Serializer):
     master_copy = EthereumAddressField(allow_null=True)
     setup_data = HexadecimalField(allow_null=True)
     data_decoded = serializers.SerializerMethodField()
-    safe_operation = aa_serializers.SafeOperationResponseSerializer(allow_null=True)
+    user_operation = aa_serializers.UserOperationWithSafeOperationResponseSerializer(
```

Comment:
This change is not indicated in the PR description, the reason behind this is that during SafeCreation the is not a SafeOperation?

Reply:
If UserOperation is reverted there's still UserOperation and noce is increased, but there's no SafeOperation as it gets reverted

Reply:
So, a SafeOperation is always linked to a UserOperation, but the opposite is not true